test/relay/relay.node.ts

```diff
@@ -192,13 +192,17 @@ describe('Dialing (via relay, TCP)', () => {
     const dialAddr = relayAddr.encapsulate(`/p2p/${relayLibp2p.peerId.toString()}`)
 
     const connection = await srcLibp2p.dial(dialAddr)
-    const stream = await connection.newStream('/libp2p/circuit/relay/0.1.0')
-
-    await stream.sink(async function * () {
-      // delay for longer than the timeout
-      await delay(1000)
-      yield Uint8Array.from([0])
-    }())
+    // this should succeed as the timeout is only effective after
+    // multistream select negotiates the protocol
+    const stream = await connection.newStream([RELAY_V2_HOP_CODEC])
+
+    /* eslint-disable-next-line no-warning-comments */
```

Comment:
Please do not disable linting

Reply:
This is a flaky test that I'm trying to solve. Would it be okay to leave the TODO comments in this case?

Reply:
TODOs are fine 👍